### PR TITLE
Add jt dis command to assist in disassembly of bitcode

### PR DESCRIPTION
**Pull request by bjfish**
Would this be useful to anyone else to include in `jt`?

### CODE REVIEW

**Comment by chrisseaton** [comment]
There is also https://github.com/graalvm/truffleruby/blob/e8cdd1aa30b2edddbc5bfa68682f1ace07c82fe5/tool/cext-compile-explore.rb which does a lot more. I'll add this to the documentation.

File: tool/jt.rb [hunk 1]
```diff
@@ -554,6 +555,15 @@ def clean
     mx(JRUBY_DIR, 'clean') if Utilities.mx?
     mvn 'clean'
   end
+  
+  def dis(*args)
+    raise ArgumentError, "expected one arg: <filename>" unless args.size == 1
```
Comment (eregon) [inline tool/jt.rb]: You could just declare it as `dis (file)`

File: tool/jt.rb [hunk 2]
```diff
@@ -554,6 +555,15 @@ def clean
...
+    editor = ENV['JT_EDITOR'] || 'vi'
+    file = `find #{JRUBY_DIR} -name "#{args[0]}"`
+    raise ArgumentError, "file not found:`#{args[0]}`" if file.empty?
+    `llvm-dis-3.8 #{file}`
+    system(editor, file.split('.')[0] + ".ll")
```
Comment (eregon) [inline tool/jt.rb]: I would just `puts` the resulting disassembled filename and then:
```
vi `jt dis FILE`
```

File: tool/jt.rb [hunk 3]
```diff
@@ -554,6 +555,15 @@ def clean
     mx(JRUBY_DIR, 'clean') if Utilities.mx?
     mvn 'clean'
   end
+  
+  def dis(*args)
+    raise ArgumentError, "expected one arg: <filename>" unless args.size == 1
+    editor = ENV['JT_EDITOR'] || 'vi'
```
Comment (eregon) [inline tool/jt.rb]: Use the standard `EDITOR` variable.

**Pull request merged**